log_request_id/session.py

```diff
@@ -24,9 +24,9 @@ def prepare_request(self, request):
         try:
             request_id = local.request_id
         except AttributeError:
-            request_id = NO_REQUEST_ID
+            request_id = DEFAULT_NO_REQUEST_ID
```

Comment:
Should this not be using the user-overidable value? Something like `getattr(settings, LOG_REQUESTS_NO_SETTING, DEFAULT_NO_REQUEST_ID)`

Reply:
Yes this is true, i dont use this part, but youre right. I'll change it.

Reply:
Ok done, i should probably add a test, let me see where to put it.

Reply:
Turns out there was no need for this, NO_REQUEST_ID is excluded from being sent anyways. I reworked it.